### PR TITLE
Fixes #11894: Saving a technique in the technique editor in IE11 fails

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/11894

### CODE REVIEW

File: builder/index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <link href="css/ngToast.min.css" rel="stylesheet">
         <link href="css/custom.css" rel="stylesheet">
         <!-- Needed polyfill for ie11, works in all other browsers -->
-        <script src="js/startswith.js"></script>
```
Comment (VinceMacBuche) [inline builder/index.html]: Can you remove file startswith.js if it is not used anymore ??

File: builder/js/ncf.js [hunk 2]
```diff
@@ -1014,8 +1015,11 @@ $scope.groupMethodsByCategory = function () {
         function(m) { 
           usedMethods.add($scope.generic_methods[m.method_name]);
         }
-      )
-      $http.post("/rudder/secure/api/latest/ncf",{"technique": ncfTechnique,"methods":Object.values($scope.generic_methods)});
+      );
+      var methodKeys = Object.keys($scope.generic_methods).map(function(e) {
```
Comment (VinceMacBuche) [inline builder/js/ncf.js]: Why n o object.values ?

File: builder/index.html (hunk 1 above)
Comment (VinceMacBuche) [reply builder/index.html]: Ok saw the rename !! great

File: builder/js/ncf.js (hunk 2 above)
Comment (RaphaelGauthier) [reply builder/js/ncf.js]: Because IE11 doesn't support Object.values... :disappointed:

Comment (VinceMacBuche) [reply builder/js/ncf.js]: Sadness :(

**Comment by RaphaelGauthier** [comment]
OK, merging this PR

**Pull request merged**